docs/faq.md

```diff
@@ -560,3 +560,22 @@ In `/var/www/html/Lychee/.env`, change `DB_HOST=localhost` to `DB_HOST=127.0.0.1
 Lychee checks for the presence of certain files to detect whether Lychee was installed as a git repository or from a release archive (.zip). It also uses them to determine which git commit is in use where applicable. This is included in the Diagnostics page to assist us in diagnosing issues.
 
 These messages can be safely ignored.
+
+### Import from server via symlink is failing.
+
+If you check the logs and see an output like this:
+
+```
+2021-11-09 19:09:50 -- error -- App\Actions\Import\Exec::do -- 256 -- Could not import file (<path-to-image>/image.jpg): 0 symlink(): Operation not supported
+```
+
+This means the native low-level function `symlink` fails, because the file system does not support symbolic links.
```

Comment:
Strictly speaking this is not 100% correct wrt. punctuation. The colon at the end of the 1st line terminates the sentence. But there is no complete sentence as the sentence goes on after the quote which also means that "This" after must be start with a small letter.

So either

````suggestion
If you check the logs and see an output like

```
2021-11-09 19:09:50 -- error -- App\Actions\Import\Exec::do -- 256 -- Could not import file (<path-to-image>/image.jpg): 0 symlink(): Operation not supported
```

means that the native low-level function `symlink` fails, because the file system does not support symbolic links.
````

or

````suggestion
Check the logs whether you see an output like this:

```
2021-11-09 19:09:50 -- error -- App\Actions\Import\Exec::do -- 256 -- Could not import file (<path-to-image>/image.jpg): 0 symlink(): Operation not supported
```

In that case the native low-level function `symlink` fails, because the file system does not support symbolic links.
````

Reply:
I don't like the second option. IMO the first needs an "it" to replace the "This".

Reply:
Ok, then the first option it is. Of course with the missing "it" or "this" in place.

Reply:
Fixed by https://github.com/LycheeOrg/LycheeOrg.github.io/pull/61/commits/dda94b818edb967874627448b7a77928a00be611